### PR TITLE
Form base#2

### PR DESCRIPTION
**Pull request by LindsayYoung**
Taking a first stab at porting over the form. It won't be pretty.

So far, I have a dummy, multi-step form and a model. The next step is to tie the form to the model.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
I made the models flat to get the sample form going quickly. I think almost everything is a drop down the multiple selects will come from many-to-many models that we can add in a following PR.

**Comment by LindsayYoung** [comment]
I will resolve the merge on package-lock

**Comment by LindsayYoung** [comment]
I see I am still missing a label, I can add that now

**Comment by alexsoble** [comment]
When I try to run `docker-compose up` against this branch, I get the following error:

```
ModuleNotFoundError: No module named 'django-formtools'
```

I get the same error even after running a docker build process and then re-running `docker-compose up`.

**Comment by alexsoble** [comment]
Pairing with @LindsayYoung and trying:

```
docker-compose run web pipenv install django-formtools
```

**Comment by alexsoble** [comment]
☝️After that step, the app works for me locally!

**Comment by alexsoble** [comment]
@LindsayYoung Is this ready for PR review, or do you want to implement checkboxes / radioboxes and other elements of the form flow before reviewing?

**Comment by alexsoble** [comment]
Lindsay and I are syncing up on this! 

Lindsay would like a review, with focus on the following areas:

- Are the main form elements in place?
- Does the overall structure make sense to me? 
- What do we need to tackle next? 

We know the form is still far from finished — there are no many-to-many relationships set up, meaning all of the questions that will eventually use checkboxes are currently showing dropdown menus instead.

But this is meant as a starting point only.

**Comment by alexsoble** [comment]
# Screenshots (fake data)

These screenshots show the basic elements of the form, which can collect data from the inputs, save to the database, and show the results in a confirmation page. Not all the form elements are the right type (i.e. checkbox, radiobox) but that's a known issue that @LindsayYoung is working on resolving. 

step-1
step-2
step-3
step-4
step-5
step-form-complete

**Comment by alexsoble** [comment]
I'm noticing that the AccessLint bot is hanging — it raised an issue, @LindsayYoung addressed the issue, and instead of giving us a green check, it says "Pending." 

I haven't used AccessLint before. Wondering if it's preferred over pa11y for a11y-based integration. Also not sure if there's some sort of AccessLint UI we can use to re-start the build and get a green checkmark.

**Comment by alexsoble** [comment]
I'm noticing that the AccessLint bot is hanging — it raised an issue, @LindsayYoung addressed the issue, and instead of giving us a green check, it says "Pending." 

I haven't used AccessLint before. Wondering if it's preferred over pa11y for a11y-based integration. Also not sure if there's some sort of AccessLint UI we can use to re-start the build and get a green checkmark.

**Comment by alexsoble** [comment]
Hmmm, closing and re-opening the PR didn't kick off a new AccessLint build.

**Comment by LindsayYoung** [comment]
@alexsoble I haven't used AccessLint before. It does seem to hang. I am open to suggestions for alternatives if you or @Jacklynn know of something better

**Comment by alexsoble** [comment]
@LindsayYoung Given that it caught an issue, that we may end up using a different tool, and that a lot of the front-end will change, I think it's OK if we ship despite the pending status — just wanted to mention it.

File: crt_portal/cts_forms/views.py [hunk 1]
```diff
@@ -1,46 +1,31 @@
...
-        # Always return an HttpResponseRedirect after successfully dealing
-        # with POST data. This prevents data from being posted twice if a
-        # user hits the Back button.
-        return HttpResponseRedirect(reverse('crt_forms:results', args=(report.id,)))
+### Forms view
 
-class DetailView(generic.DetailView):
-    model = ViolationReport
-    template_name = 'forms/detail.html'
+class ContactWizard(SessionWizardView):
```
Comment (alexsoble) [inline crt_portal/cts_forms/views.py]: @LindsayYoung Not necessarily a blocker, but do we want to rename this at some point to something like `CivilRightsComplaintWizard`?

File: crt_portal/cts_forms/urls.py [hunk 2]
```diff
@@ -6,8 +6,7 @@
 app_name = 'crt_forms'
 
 urlpatterns = [
-    path('', views.IndexView.as_view(), name='index'),
-    path('<int:pk>/', views.DetailView.as_view(), name='detail'),
-    path('<int:pk>/results/', views.ResultsView.as_view(), name='results'),
-    path('<int:violationreport_id>/vote/', views.VoteView, name='vote'),
+    # path('', views.IndexView.as_view(), name='index'),
```
Comment (alexsoble) [inline crt_portal/cts_forms/urls.py]: @LindsayYoung No URLs for `cts_forms`?

Comment (LindsayYoung) [reply crt_portal/cts_forms/urls.py]: We haven't made them yet, the ones we took away were from the prototype.

Comment (alexsoble) [reply crt_portal/cts_forms/urls.py]: @LindsayYoung I see! Without any code comments the intention here isn't clear. I think a code comment about the intention would be helpful, but not necessarily a blocker.

**Comment by alexsoble** [comment]
Looks good locally to me!

**Pull request merged**